frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -190,7 +190,7 @@ export const NavBar = withRouter<NavBarProps>(({ pages, baseURL, basePath }) =>
           'co-m-horizontal-nav-item--active': matchURL?.isExact,
         });
         return (
-          <li className={klass} key={nameKey || name}>
+          <li className={klass} key={nameKey || `${name}${index}`}>
```

Comment:
Not a fan of using the index as a key and couldn't find a unique identifier.  But I just realized that the url could be used instead.  This code will be changed shortly.

Reply:
Changed to use href instead of array index.